src/main/getSessionInfo.spec.ts

```diff
@@ -0,0 +1,45 @@
+import fetchMock from "jest-fetch-mock"
```

Comment:
This file should be in the `tests` folder

Reply:
Nice catch 🤦